src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let number = 0;
+
+  if (days < 3) {
+    number = days * 40;
+  } else if (days >= 3 && days < 7) {
+    number = days * 40 - 20;
+  } else if (days >= 7) {
+    number = days * 40 - 50;
```

Comment:
This `else if` block can be replaced with a simple `if` since the previous conditions would have already returned if met. This simplifies the flow of the function.